config.py

```diff
@@ -5,4 +5,5 @@
 CREDENTIALS_PATH = join(GDRIVE_PATH, 'credentials.json')
 TOKEN_PATH = join(GDRIVE_PATH, 'token.pickle')
 EXTRACTOR_CONFIG_FILE = join(GDRIVE_PATH, 'data_config.json')
-
+UPLOAD_CHUNK_SIZE = 1024 * 1024 * 10  # 10MB
+DOWNLOAD_CHUNK_SIZE = 1024 * 1024 * 10  # 10MB
```

Comment:
What are your thoughts about these values for different internet speeds? And why did you pick these values?

Reply:
IQQ. Please run some tests with your setup. We might benefit from some logic to infer the chunk size optimally.